src/Permission.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Storj\Uplink;
+
+use DateTimeImmutable;
+use FFI;
+use FFI\CData;
+
+/**
+ * Parameters to create an Access grant
+ */
+class Permission
```

Comment:
Can we have method changing to allow something like:
```php
$permission = new Permission()->allowUpload()->notAfter(2hours);
```

Reply:
Also, we should have the equivalent of these:
- https://pkg.go.dev/storj.io/uplink?tab=doc#FullPermission
- https://pkg.go.dev/storj.io/uplink?tab=doc#ReadOnlyPermission
- https://pkg.go.dev/storj.io/uplink?tab=doc#WriteOnlyPermission

Perhaps as static methods?

Reply:
Added the static methods and removed the constructor

```php
$permission = (new Permission())->allowUpload()->notAfter(new DateTimeImmutable('+2 hours'));
```